sdk/search/search-documents/src/serviceModels.ts

```diff
@@ -1133,10 +1138,12 @@ export enum KnownTokenizerNames {
   /**
    * Divides text using language-specific rules.
    */
+  // eslint-disable-next-line @typescript-eslint/no-shadow
```

Comment:
the linter considers Enum names to be shadowing? Can we file a bug on this?

Reply:
The rule can be configured as follows: https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-shadow.md

Reply:
@xirzec Yes. I will file a bug for this with linter.

Reply:
Or does the ignoreTypeValueShadow setting solve this?

Reply:
I think this will ignore it for the whole project.